### PR TITLE
Add hint to call supports in getCredentials method for symfony lower than 3.4

### PR DESCRIPTION
**Pull request by alexander-schranz**
With https://github.com/knpuniversity/oauth2-client-bundle/pull/79 the docs is not longer working for lower than symfony 3.4. So a hint should be added how to get it work with <3.4

### CODE REVIEW

**Comment by weaverryan** [comment]
Thanks @alexander-schranz! I think it's actually 3.3 and lower that have the issue - 3.4 should act like 4.0. But, I like the idea - we just need to tweak the message.

**Comment by alexander-schranz** [comment]
@weaverryan updated the message to lower than 3.4 and fix the supports function call.

**Comment by larzuk91** [comment]
It looks good to me :+1:

File: README.md [hunk 1]
```diff
@@ -273,6 +273,11 @@ class MyFacebookAuthenticator extends SocialAuthenticator
     {
         // this method is only called if supports() returns true
 
+        // For Symfony lower than 3.4 the supports method need to be called manually here:
+        // if (!$this->supports($request)) {
+        //    return;
```
Comment (bocharsky-bw) [inline README.md]: Let's explicitly return `null` here since this method is supposed to return something, but just `return;` is supposed to return `void`

**Comment by weaverryan** [comment]
Thank you again!

**Pull request merged**